support-files/mariadb.logrotate.in

```diff
@@ -0,0 +1,56 @@
+# This is the MariaDB configuration for the logrotate utility
+#
+# Note that on most Linux systems logs are written to journald, which has its
+# own rotation scheme.
+#
+# Read https://mariadb.com/kb/en/error-log/ to learn more about logging and
+# https://mariadb.com/kb/en/rotating-logs-on-unix-and-linux/ about rotating logs.
+
+@localstatedir@/mysqld.log @logdir@/mysql.log @localstatedir@/mariadb.log @logdir@/mysql-slow.log @logdir@/mariadb-slow.log @logdir@/error.log {
```

Comment:
@ottok I guess there is a problem here.

Somehow, `@logdir@` is empty on Debian and causing the logrotate service to fail. I have just reproduced https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=987231.

Reply:
Ok, as commented by Matthew in #987231 the actual version is just missing `SET(logdir "/var/log/mysql")` that is defined [here](https://github.com/MariaDB/server/pull/1556/files#diff-4bfceb03afa373d0cf49ad955b3a1801decfc9048f43b4731ecf1fab38afb6c0R36).